components/src/core/Drawer/DrawerRailItem/DrawerRailItem.tsx

```diff
@@ -68,109 +82,137 @@ export type DrawerRailItemProps = SharedStyleProps & {
+        minHeight: RAIL_WIDTH,
+        position: 'relative',
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        justifyContent: 'center',
+        cursor: 'default',
+        padding: `1rem ${statusColor ? theme.spacing(1) : theme.spacing(0.5)}px`,
+        textAlign: 'center',
+        backgroundColor: backgroundColor || 'transparent',
+        '&:hover': {
+            backgroundColor: onClick ? theme.palette.action.hover : undefined,
         },
-        active: {
-            position: 'absolute',
-            top: 0,
-            left: 0,
-            right: 0,
-            bottom: 0,
-            backgroundColor: (props): string =>
-                props.activeItemBackgroundColor ||
-                (theme.palette.mode === 'light' ? fivePercentOpacityPrimary : twentyPercentOpacityPrimary),
-        },
-        condensed: {
-            width: RAIL_WIDTH_CONDENSED,
-            minHeight: RAIL_WIDTH_CONDENSED,
-        },
-        cursorPointer: {
+        [`& .${drawerRailItemClasses.cursorPointer}`]: {
             cursor: 'pointer',
         },
-        divider: {
-            position: 'absolute',
-            bottom: 0,
-            left: 0,
-            width: '100%',
-        },
-        icon: {
-            color: (props): string => props.itemIconColor || theme.palette.text.primary,
-            backgroundColor: 'transparent',
-            height: 'auto',
-            width: 'auto',
-            overflow: 'visible',
-        },
-        itemActive: {
+        [`& .${drawerRailItemClasses.itemActive}`]: {
             '& $icon': {
-                color: (props): string =>
-                    props.activeItemIconColor ||
+                color:
+                    activeItemIconColor ||
                     (theme.palette.mode === 'light' ? theme.palette.primary.main : lightenedPrimary),
             },
             '& $title': {
-                color: (props): string =>
-                    props.activeItemFontColor ||
+                color:
+                    activeItemFontColor ||
                     (theme.palette.mode === 'light' ? theme.palette.primary.main : lightenedPrimary),
             },
         },
-        ripple: {
-            backgroundColor: theme.palette.primary.main,
-        },
-        statusStripe: {
-            position: 'absolute',
-            top: 0,
-            bottom: 0,
-            left: 0,
-            width: 6,
-            zIndex: 100,
-            backgroundColor: (props): string => props.statusColor,
-        },
-        title: {
-            lineHeight: '1rem',
-            wordBreak: 'break-word',
-            hyphens: 'auto',
-            zIndex: 200,
-            color: (props): string => props.itemFontColor || theme.palette.text.primary,
+        [`& .${drawerRailItemClasses.condensed}`]: {
+            width: RAIL_WIDTH_CONDENSED,
+            minHeight: RAIL_WIDTH_CONDENSED,
         },
-        titleActive: {
```

Comment:
Where did this class go?

Reply:
titleActive class has been added to Title styled component